ci/helm/values/values-mongodb.yaml.gotmpl

```diff
@@ -0,0 +1,26 @@
+image:
+  tag: {{ .Values.mongodb.image.tag }}
+usePassword: false
+serviceAccount:
+  create: false
+resources:
```

Comment:
You might want to disable persistence, it's enabled by default in the latest chart: https://github.com/bitnami/charts/blob/master/bitnami/mongodb/values.yaml#L792
(to be checked in tha actual version of the chart)

Reply:
Let's disable it to avoid any surprise on upgrades.